python/ray/train/torch/torch_checkpoint.py

```diff
@@ -45,15 +52,48 @@ def from_model(
             >>> model = torch.nn.Identity()
             >>> checkpoint = TorchCheckpoint.from_model(model)
 
-            You can use a :py:class:`TorchCheckpoint` to create an
-            :py:class:`~ray.train.torch.TorchPredictor` and preform inference.
+            You can use a :class:`TorchCheckpoint` to create an
+            :class:`~ray.train.torch.TorchPredictor` and perform inference.
 
             >>> from ray.train.torch import TorchPredictor
             >>>
             >>> predictor = TorchPredictor.from_checkpoint(checkpoint)
+        """  # noqa: E501
+        return cls.from_dict({PREPROCESSOR_KEY: preprocessor, MODEL_KEY: model})
+
+    @classmethod
+    def from_state_dict(
+        cls, state_dict: OrderedDict, *, preprocessor: Optional["Preprocessor"] = None
```

Comment:
does state dict have to be an OrderedDict?

Reply:
Checked the `torch.nn.Module` source. It can be any `Dict[str, Any]`.

Fixed.